tests/python/test_transformer_engine.py

```diff
@@ -32,13 +47,10 @@ class ComputeType(Enum):
 @pytest.mark.mpi
 @pytest.mark.parametrize(
```

Comment:
I am not sure but it might work to specify `scope=function` here and remove the scope arguments from the two fixtures

Reply:
I'm not sure I understand your suggestion. AFAIK, `scope` is an attribute of a fixture, not of a test.